db/migrate/20200127120545_migrate_data_to_user_dishes.rb

```diff
@@ -0,0 +1,14 @@
+class MigrateDataToUserDishes < ActiveRecord::Migration[5.0]
+  def up  
+    Dish.all.pluck(:user_id, :id).each do |user_id, id|
+      UserDish.create!(user_id: user_id, dish_id: id)
+    end
+  end
+  
+  def down
+    UserDish.all.pluck(:user_id, :dish_id).each do |user_id, dish_id|
+      Dish.find(dish_id).update!(user_id: user_id)
+    end
+    UserDish.delete_all
```

Comment:
Just making sure - this will not trigger any callbacks, you know that? Only `destroy` triggers callbacks, so in this instance no `dependent: :whatever` will happen.

Reply:
There isn't anything dependent on `UserDish`, so it's working as its supposed to, we want to delete only `UserDish`es